source/jormungandr/jormungandr/parking_space_availability/car/common_car_park_provider.py

```diff
@@ -60,6 +61,15 @@ def __init__(self, url, operators, dataset, timeout, feed_publisher, **kwargs):
 
         self.api_key = kwargs.get('api_key')
 
+        self.shape = None
+        arg_geometry = kwargs.get('geometry')
+        if arg_geometry:
+            try:
+                self.shape = shapely.geometry.shape(arg_geometry)
+            except Exception as e:
+                self.log.error("Enable to parse geometry object from : {}".format(arg_geometry))
```

Comment:
Does this print something readable?

Reply:
Well, the formating might not be amazing, especially if your shape is big. But at least you have an idea that you error comes from the shape you're passing in.
I'm open for suggestion for a more friendly user error message :)